From 12282261d3f0f1556891efbcca9744268f106c8a Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Thu, 22 Jun 2023 15:58:15 +0200
Subject: [PATCH 0350/1141] arm64: dts: st: add power supply to hdmi on
 stm32mp257f-ev1 board

To avoid warnings, add power supply to hdmi bridge.

Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Change-Id: Iefbafef57328943f0cd75df63dfc2fa84a2e47e0
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/314619
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
index b9627bd75757..e9f6980aa174 100644
--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -314,6 +314,12 @@ adv7535: hdmi@3d {
 		status = "disabled";
 		adi,dsi-lanes = <4>;
 		reset-gpios = <&gpiog 14 GPIO_ACTIVE_LOW>;
+		avdd-supply = <&scmi_v3v3>;
+		dvdd-supply = <&scmi_v3v3>;
+		pvdd-supply = <&scmi_v3v3>;
+		a2vdd-supply = <&scmi_v3v3>;
+		v3p3-supply = <&scmi_v3v3>;
+		v1p2-supply = <&scmi_v3v3>;
 
 		port {
 			adv7535_in: endpoint {
-- 
2.39.2

